### PR TITLE
[Icds dashboard] remove disappear condition from sag

### PR DESCRIPTION
**Pull request by Rohit25negi**
The behavior of the SAG is changed. Now we will not hide the card, navigation menu, fact sheet indicators, tabular report indicators when going back to any time. It will just display 0s.

### CODE REVIEW

File: custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html [hunk 1]
```diff
@@ -115,14 +115,15 @@
...
+              <!-- This div item will be removed after QA -->
                 <div class="navigation-font pl-3 pr-3 mb-0"
                      ng-if="haveAccessToFeatures==false"
                      ng-class="{'active-mobile-link': $location.path().indexOf('/demographics/adolescent_girls') !== -1}">
                     <a href="{$ goToStep('demographics/adolescent_girls', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
                       Adolescent Girls enrolled for Anganwadi Services</a>
                 </div>
               <div class="navigation-font pl-3 pr-3 mb-0"
-                     ng-if="toShowOnNavigation()"
+                     ng-if="haveAccessToFeatures==true"
```
Comment (czue) [inline custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html]: nit: is `==true` necessary here?

**Review by czue — APPROVED** [review]
:+1: lgtm

File: custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html]: not really technically, but added just to make the code clearer.

Comment (czue) [reply custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html]: interesting. i think not including it is clearer, though suppose is subjective

**Pull request merged**